fw/apm.c

```diff
@@ -944,14 +951,11 @@ tfw_apm_calc(TfwApmData *data)
  * tfw_apm_stats() should be used for calls in kernel context.
  * tfw_apm_stats_bh() should be used for calls in user context.
  */
-#define __tfw_apm_stats_body(apmref, pstats, fn_lock, fn_unlock)	\
+#define __tfw_apm_stats_body(data, pstats, fn_lock, fn_unlock)		\
```

Comment:
May be change this macros to static inline function? I don't think that it is good to return  macros

Reply:
Fixed in further commit, but here we can't simply do this because of fns (lambda seem to be overengineering from my side, but maybe I'm wrong).